javascript/filter.js

```diff
@@ -0,0 +1,42 @@
+function filterSelection(section) {
+  let element = document.getElementsByClassName("portfolio__box");
+  if (section == "all") {
+    section = "";
+  }
+  for (let i = 0; i < element.length; i++) {
+    RemoveClass(element[i], "portfolio__show");
+    if (element[i].className.indexOf(section) > -1)
+      AddClass(element[i], "portfolio__show");
+  }
+}
```

Comment:
- Don't forget that you should use constants where needed.
- Try to remove the repetition 'element[i]' with something like `const currentElement = element[i]`
- You named your variable `element`, but you actually store more than one element inside of it. Maybe name it something like `elements` or `portfolioBoxes`
- You are not updating your SwiperJS instance when you add/hide slides, which results in pagination that you don't need
- ![image](https://user-images.githubusercontent.com/68795596/205644843-33761b46-34ce-4250-ba03-e598d39b9970.png)

And finally, why are you doing:
```
...
<button class="portfolio__btn" onclick="filterSelection('all')">All</button>
...
```
and then
```
if (section == "all") {
  section == ""
}
```
instead of just passing "" as a parameter

Another option and the way I would write it:

```
function filterSelection(section) {
  const elements = document.getElementsByClassName("portfolio__box");

  for (let i = 0; i < elements.length; i++) {
    const currentElement = elements[i]
    if (currentElement.classList.contains(section) || (section === 'all')) {
      AddClass(currentElement, "portfolio__show"); 
    } else {
      RemoveClass(currentElement, "portfolio__show");
    }
  }
  /* ... something that updates your Swiper instance ... */
}
```